app/models/partner.rb

```diff
@@ -2,18 +2,88 @@
 #
 # Table name: partners
 #
-#  id                          :integer          not null, primary key
-#  email                       :string
-#  name                        :string
-#  notes                       :text
-#  quota                       :integer
-#  send_reminders              :boolean          default(FALSE), not null
-#  status                      :integer          default("uninvited")
-#  created_at                  :datetime         not null
-#  updated_at                  :datetime         not null
-#  default_storage_location_id :bigint
-#  organization_id             :integer
-#  partner_group_id            :bigint
+#  id                         :bigint           not null, primary key
+#  above_1_2_times_fpl        :integer
```

Comment:
All of these generated columns should be on the `app/models/partners/partner.rb` model.

Not sure how they got put in and the next time someone runs a migration they should get fixed.

Reply:
This happens every time I run migrations for some reason. A better solution might be for the annotations to happen on a GitHub Action rather than on db:migrate locally.